user-interface/src/data-verification/consolidation/consolidationsUseCase.ts

```diff
@@ -309,12 +285,23 @@ const consolidationUseCase = (
 
   const updateSubmitButtonsState = () => {
     if (store.selectedCases.length) {
-      const disableApprove =
-        store.selectedCases.length <= 1 ||
-        !store.isDataEnhanced ||
-        store.leadCaseId === '' ||
-        store.consolidationType === null ||
-        areAnySelectedCasesConsolidated();
+      const alreadyConsolidatedLeadCase = store.selectedCases.find((bCase) =>
+        bCase.associations?.find((assoc) => assoc.documentType === 'CONSOLIDATION_FROM'),
+      );
+
+      let disableApprove = false;
+      if (alreadyConsolidatedLeadCase && store.leadCaseId !== alreadyConsolidatedLeadCase.caseId) {
+        disableApprove = true;
+      } else {
+        disableApprove =
+          store.selectedCases.length === 1 && store.leadCaseId === store.selectedCases[0].caseId;
+        disableApprove =
+          disableApprove ||
+          !store.isDataEnhanced ||
+          store.leadCaseId === '' ||
+          store.consolidationType === null ||
+          areAnySelectedCasesConsolidated();
+      }
```

Comment:
We added a decent bit of additional logic here, but didn't update / add any new tests for these new scenarios. We should consider adding additional test coverage around this.